wiki/content/deploy/tls-configuration.md

```diff
@@ -139,36 +179,70 @@ Dgraph Live Loader can be configured with the following options:
+# Next, set up Dgraph Zero nodes using the following default command:
+$ dgraph zero --tls_cacert /dgraph-tls/ca.crt --tls_node_cert /dgraph-tls/node.crt --tls_node_key /dgraph-tls/node.key
 ```
 
-You can then run Dgraph live loader using the following:
+You can then run Dgraph Live Loader on a Dgraph Alpha node using the following command:
 
 ```sh
 # Now, connect to server using TLS
-$ dgraph live --tls_cacert ./tls/ca.crt --tls_server_name "localhost" -s 21million.schema -f 21million.rdf.gz
+$ dgraph live --tls_cacert ./dgraph-tls/ca.crt --tls_server_name "localhost" -s 21million.schema -f 21million.rdf.gz
 ```
 
-### Using TLS with Client Authentication
+### Using TLS with client authentication (encrypt internal and external ports)
+
+If you require client authentication (mutual TLS, or mTLS), you can configure certificates and run Dgraph Alpha and Dgraph Zero with settings that encrypt both internal ports (those used within the cluster) as well as external ports (those used by clients that connect to the cluster).
 
-If you do require Client Authentication (Mutual TLS), you can configure certificates and run Alpha server using the following:
+The following example shows how to encrypt both internal and external ports. The
+`REQUIREANDVERIFY` client authentication option is required if internal ports
+are encrypted:
+
+Dgraph Alpha:
 
 ```sh
-# First, create a rootca, node, and client certificates and private keys
-$ dgraph cert -n localhost -c dgraphuser
-# Default use for enabling TLS server with client authentication (after generating certificates and private keys)
-$ dgraph alpha --tls_dir tls --tls_client_auth="REQUIREANDVERIFY"
+# First create the root CA, node certificates and private keys, if not already created.
+# Note that you must specify the host name or IP address for other nodes that will share node.crt.
+$ dgraph cert -n localhost,104.25.165.23
+# Set up Dgraph Alpha nodes using the following default command (after generating certificates and private keys)
+$ dgraph alpha
+      --tls_cacert /dgraph-tls/ca.crt --tls_node_cert /dgraph-tls/node.crt --tls_node_key /dgraph-tls/node.key
+      --tls_internal_port_enabled=true --tls_cert /dgraph-tls/client.alpha1.crt --tls_key /dgraph-tls/client.alpha1.key
+      --tls_client_auth="REQUIREANDVERIFY"
+```
+
+Dgraph Zero:
+
+```sh
+# First, copy the certificates and private keys used to set up Dgraph Alpha (above) to the Dgraph Zero node.
+# Next, set up Dgraph Zero nodes using the following default command:
+$ dgraph zero
+      --tls_cacert /dgraph-tls/ca.crt --tls_node_cert /dgraph-tls/node.crt --tls_node_key /dgraph-tls/node.key
+      --tls_internal_port_enabled=true --tls_cert /dgraph-tls/client.zero1.crt --tls_key /dgraph-tls/client.zero1.key
+      --tls_client_auth="REQUIREANDVERIFY"
 ```
 
-You can then run Dgraph live loader using the following:
+You can then run Dgraph Live Loader using the following:
 
```

Comment:
based on tls client option as REQUIREANDVERIFY, live loader config needs to be updated. Since Internal ports are also tls encrypted, please add this `--tls-internal-port-enabled=true`